roxie/ccd/ccdlistener.cpp

```diff
@@ -1566,12 +1566,8 @@ class RoxieProtocolMsgContext : implements IHpccProtocolMsgContext, public CInte
 
         ensureContextLogger();
 
-        const char * spanQueryName = !isEmptyString(queryName) ? queryName : "run_query";
-        StringBuffer spanName(querySetName);
-        if (spanName.length())
-            spanName.append('/');
-        spanName.append(spanQueryName);
-        requestSpan.setown(queryTraceManager().createServerSpan(spanName, allHeaders, flags));
+        requestSpan.setown(queryTraceManager().createServerSpan(!isEmptyString(queryName) ? queryName : "run_queryX", allHeaders, flags));
```

Comment:
I presume intentional that if queryName undefined it is now "run_queryX" (extra X) vs previously "run_query" ?

Reply:
Great catch @jakesmith . No that was used in testing. I'll revert.